redis_enterprise/manifest.json

```diff
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b569beaa-dbf6-4c40-a640-fab0ea2b9cab",
+  "app_id": "redis-enterprise",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Redis Enterprise Datadog Integration",
+    "title": "Redis Enterprise v2",
```

Comment:
@j8-redis This is the title of the tile. Do you want to keep Redis Enterprise v2 or change this to Redis Enterprise?